app/(main)/league/[leagueId]/entry/[entryId]/week/WeekTeams.test.tsx

```diff
@@ -94,4 +103,26 @@ describe('WeekTeams', () => {
     expect(packersButton).toBeInTheDocument();
     expect(packersButton).toBeDisabled();
   });
+
+  describe('team loading spinner', () => {
+    it('should show the loading spinner', async () => {
+      jest.fn().mockImplementationOnce(() => ['packers', setTeamLoadingName]);
+
+      render(<TestWeekTeamsComponent />);
+
+      await waitFor(() => {
+        expect(screen.queryByTestId('loading-spinner')).not.toBeInTheDocument();
```

Comment:
This should be testing that the loading spinner _is_ in the document. However, your test is checking that is is _not_ in the document.

Reply:
Thank you for catching this. Must have missed that 😅

Reply:
LGTM